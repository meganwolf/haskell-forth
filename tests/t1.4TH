84 EMIT 104 EMIT 105 EMIT 115 EMIT 32 EMIT 105 EMIT 115 EMIT 32 EMIT 109 EMIT 121 EMIT 32 EMIT 69 EMIT 77 EMIT 73 EMIT 84 EMIT 32 EMIT 102 EMIT 117 EMIT 110 EMIT 99 EMIT 116 EMIT 105 EMIT 111 EMIT 110 EMIT 97 EMIT 108 EMIT 32 EMIT 116 EMIT 101 EMIT 115 EMIT 116 EMIT 46 EMIT 032 EMIT 73 EMIT 102 EMIT 32 EMIT 121 EMIT 111 EMIT 117 EMIT 32 EMIT 97 EMIT 114 EMIT 101 EMIT 32 EMIT 114 EMIT 101 EMIT 97 EMIT 100 EMIT 105 EMIT 110 EMIT 103 EMIT 32 EMIT 116 EMIT 104 EMIT 105 EMIT 115 EMIT 44 EMIT 32 EMIT 105 EMIT 116 EMIT 32 EMIT 119 EMIT 111 EMIT 114 EMIT 107 EMIT 115 EMIT 046 EMIT