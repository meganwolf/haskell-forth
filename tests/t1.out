This is my EMIT functional test. If you are reading this, it works.